docs/getting-started.md

```diff
@@ -3,6 +3,48 @@ sidebar_position: 1
 slug: /
 ---
 
-# Getting Started
+## How to Build a Guided Project
 
-Welcome to the **Author Workbench Knowledgebase**. We are in the process of building this knowledgebase to help you build **amazing courses and guided projects** with Author Workbench.
+Congratulations on creating your Guided Project. This article will walk you through the steps to create and publish a Guided Project.
+
+## Prerequisites
+
+If you haven't went through our onboarding process, please visit https://www.skills.network/authoring/ first and follow the instructions.
```

Comment:
This URL is an old one, it's https://skills.network/authors now. I also think we can remove this section as I don't see much value in it. Anyone on this page will have already gone through onboarding